pages/k8s/upgrade-notes.md

```diff
@@ -277,8 +277,8 @@ Once you have a Containerd backed Charmed Kubernetes running, you can add Docker
 workers like so:
 
 ```bash
-juju deploy cs:~containers/kubernetes-worker kubernetes-worker-docker
-juju deploy cs:~containers/docker
+juju deploy kubernetes-worker kubernetes-worker-docker
+juju deploy docker
```

Comment:
as above

Reply:
it's perhaps worth noting that the "upgrade-notes" section you're in with these comments is `Upgrading to 1.15`, so it's not like we're recommending these steps with a current release.  i'd be +1 to drop anything < 1.21 from this page if you'd like, but if we're gonna keep these historical entries, i think we should keep them updated with the current syntax.

Reply:
ok